Sources/swift-parsing-benchmark/Arithmetic.swift

```diff
@@ -10,35 +10,59 @@ import Parsing
 
 // MARK: - Parsers
 
-private let additionAndSubtraction = InfixOperator(
-  OneOfMany(
-    "+".utf8.map { (+) },
-    "-".utf8.map { (-) }
-  ),
-  associativity: .left,
-  lowerThan: multiplicationAndDivision
-)
+private struct AdditionAndSubtraction: Parser {
```

Comment:
We can eliminate `Lazy` and `AnyParser` in the arithmetic parsing benchmark by using proper types.